website/content/en/events/2023-kubecon-eu.md

```diff
@@ -1,6 +1,7 @@
 ---
 title: Kubecon CloudNativeCon EU 2023
 description: TAG Environmental Sustainability presence at Cloud Native Computing Foundation’s flagship conference in Amsterdam, The Netherlands from 18-21 April, 2023.
+weight: 5
```

Comment:
we may want to use a very high value like 99 so we dont need to change all the weights with every events page we add, and just count down.

Reply:
I think that is a sensible option. Do you envision always having all the past events? I think what I mentioned about a past/upcoming would be good in the future.

Reply:
I think it's most natural to list the events in chronological order as they happen.